plugins/poll/plugin.rb

```diff
@@ -367,6 +368,12 @@ def extract(raw, topic_id, user_id = nil)
             poll["options"] << { "id" => option_id, "html" => o.inner_html.strip }
           end
 
+          # title
+          title_element = p.css(".poll-title").first
+          if title_element
```

Comment:
In Ruby, you can merge both lines

```suggestion
          if title_element = p.css(".poll-title").first
```